app/Services/ProductAttributeValue.php

```diff
@@ -926,6 +933,8 @@ protected function prepareEntity(Entity $entity): void
         $entity->set('prohibitedEmptyValue', $attribute->get('prohibitedEmptyValue'));
         $entity->set('attributeGroupId', $attribute->get('attributeGroupId'));
         $entity->set('attributeGroupName', $attribute->get('attributeGroupName'));
+        $this->setUseDisabledTextareaInViewMode($entity, $attribute->get('useDisabledTextareaInViewMode'));
```

Comment:
No needs to write additional function here. Just write your logic here in this function.